test/pj-rehearse-integration/candidate/ci-operator/config/super/duper/super-duper-ciop-cfg-change.yaml

```diff
@@ -0,0 +1,28 @@
+base_images:
```

Comment:
Did the addition of this data impact the integration test at all?

Reply:
Very little. It just causes the new code to be executed. So we catch panics and test only that current observable (sans new log messages) behavior does not change by the presence of ciop config and new code.